api-reference/10 UI Components/GridBase/1 Configuration/columnAutoWidth.md

```diff
@@ -10,9 +10,9 @@ Specifies whether columns should adjust their widths to the content.
 ---
 When this property is set to **true**, all columns adjust their width to the content.
 
-If the UI component's overall content is narrower than the UI component's width, the columns are stretched to fit the UI component. To avoid this, set the **columns**.[width](/api-reference/_hidden/GridBaseColumn/headerFilter/width.md '/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/headerFilter/#width') property to *"auto"*.
+If the {WidgetName} is wider that its overall content, the columns are stretched to occupy all available width. To avoid this, set the **columns**.[width]({basewidgetpath}/Configuration/columns/headerFilter/#width) property to *"auto"*.
```

Comment:
```suggestion
If the {WidgetName} is wider than its overall content, the columns are stretched to occupy all available width. To avoid this, set the [columnWidth]({basewidgetpath}/Configuration//#columnWidth) property to *"auto"*.
```